src/tests/ftest/get_remote_files.sh

```diff
@@ -180,12 +180,15 @@ list_tag_files() {
 #   1: No VM detected or skipped
 #######################################
 check_hw() {
-    if [ "${1}" == "true" ]; then
-        echo "  Skipping VM check for compression"
-        return 1
-    fi
     dmesg | grep "Hypervisor detected" >/dev/null 2>&1
-    return $?
+    ret=$?
+    if [ ${ret} -eq 0 ]; then
+        if [ "${1}" == "true" ]; then
+            echo "  Running compression on a VM host"
+            ret=1
+        fi
+    fi
+    return ${ret}
```

Comment:
In conjunction with the above:

```suggestion
    return 0
```